Modules/booking/Controller/BookingperiodicController.php

```diff
@@ -457,46 +450,27 @@ private function isNew($param) {
     }
 
     public function deleteAction($id_space, $id){
+        $sendEmail = $this->request->getParameter("sendmail");
         
-        $sendmail = $this->request->getParameter("sendmail");
-        
-        if( $sendmail == 1){
+        if( $sendEmail == 1){
```

Comment:
does it works with php8 (sendmail will be a string no?)

Reply:
Yes sendemail is a string and this works in php8.